### PR TITLE
[Bugfix] Scaleline values are constraint by default baselayer and not the map

### LINKED ISSUE #2978: weird zoom behavior - zoom stops at 14076 value in the scale
### What is the bug?

zoom stops at 14076 value in the scale!

lizmap_

in all my project (projection 3763) the map scales are different between 3.4 and 3.5 lwc versions

3.4 - 3.5

141 - ???
282 - ???
564 - ???
1128 - ???
2257 - ???
4514 - ???
9028 - ???
18056 - 14076 - zoom STOPs here
36112 - 28152
72224 - 56304
144448 - 112608
288895 - 225217
577791 - 450433

### Steps to reproduce the issue

try a deep zoom `by rectangule` over visible features, and `zoom in`, `zoom in`...
or
try from initial map extent `zoom in`, `zoom in`, `zoom in`, ...

i can provide an url if you want to inspect this weird zoom behavior

thanks

### Versions

Lizmap Web Client : 3.5.3
Lizmap plugin : 3.7.7
QGIS Desktop : 3.22.6

### QGIS server version, only if the section above doesn't mention the QGIS Server version

3.22.6

### Operating system

Ubuntu 20.04

### Browsers

Firefox

### Browsers version

99.0.1

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by rldhont**
Fix #2978  weird zoom behavior - zoom stops at 14076 value in the scale

Funded by 3liz

### CODE REVIEW

**Comment by rldhont** [comment]
end2end needs to be updated before merging

**Comment by rldhont** [comment]
Ready for review

File: tests/qgis-projects/tests/base_layers.qgs.cfg [hunk 1]
```diff
@@ -17,14 +17,15 @@
...
+            1000,
+            5000,
             25000,
             50000,
             100000,
             250000,
             500000
         ],
-        "minScale": 10000,
+        "minScale": 500,
```
Comment (nboisteault) [inline tests/qgis-projects/tests/base_layers.qgs.cfg:28]: `minScale` is not equal to `mapScales` minimum?

Comment (rldhont) [reply tests/qgis-projects/tests/base_layers.qgs.cfg:28]: Thnaks for the review

**Review by mdouchin — APPROVED** [review]
👍

**Pull request merged**